app/assets/stylesheets/_functions.scss

```diff
@@ -0,0 +1,42 @@
+@function rd($x, $y) {
+  $size: percentage($x / $y);
+  @return $size;
+}
+@function rem($x) {
+  $size: ($x / 16);
+  @return #{$size}rem;
+}
+@mixin fontsize($size) {
+  font-size: #{$size}px; //Fallback in px
+  font-size: rem($size);
+}
+
+///
+$max-vw: 900px;
+$min-vw: 600px;
+
+@mixin fluid-size($properties, $min-value, $max-value) {
+  @each $property in $properties {
+    #{$property}: $min-value;
+  }
+
+  @media screen and (min-width: $min-vw) {
+    @each $property in $properties {
+      #{$property}: calc(#{$min-value} + #{strip-unit($max-value - $min-value)} * (100vw - #{$min-vw}) / #{strip-unit($max-vw - $min-vw)});
+    }
+  }
+
+  @media screen and (min-width: $max-vw) {
+    @each $property in $properties {
+      #{$property}: $max-value;
+    }
+  }
+}
+
+@function strip-unit($value) {
+  @return $value / ($value * 0 + 1);
+}
+
+// p {
+//   @include fluid-size('font-size', 16px, 32px);
+// }
```

Comment:
Should this comment be here? If its not needed, I would think it should be deleted.